error-issue/src/service_error_monitor.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2020 The AMP HTML Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {ErrorMonitor} from './error_monitor';
+import {ServiceGroup, Stackdriver} from 'error-issue-bot';
+import {StackdriverApi} from './stackdriver_api';
+
+export enum ServiceName {
+  PRODUCTION = 'CDN Production',
+  DEVELOPMENT = 'CDN Development',
```

Comment:
Can these be renamed to the channel names? i.e. STABLE, BETA

Reply:
These are larger groupings than just the channels. "CDN Production" includes LTS, Stable, and Control channels. "CDN Development" includes Beta and Experimental. "CDN Experiments" includes Experiment-[ABC]. The idea here is that, when looking at and measuring frequency of errors, we can group all versions that are running the same commit of the code. In other words, they correspond to the Service bucket (first dropdown), not the actual Version (second dropdown)